chrome/content/zotero/xpcom/date.js

```diff
@@ -477,6 +477,48 @@ Zotero.Date = new function(){
 		return date;
 	}
 	
+	this.isHTTPDate = function(str) {
+		var dayNames = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'];
+		var monthNames = this.getMonths().short;
```

Comment:
Updated. I trust this code I wrote and I don't trust a regex without enough approval/disapproval. So this code it is.